static/src/javascripts/projects/common/modules/sport/football/match-info.js

```diff
@@ -0,0 +1,29 @@
+import fetch from 'lib/fetch';
+
+/**
+ * @param {Object} match
+ * @param {string} whosCalling (url)
+ */
+
+class MatchInfo {
+    constructor(match, whosCalling) {
+        const urlBase = '/football/api/match-nav';
+        const pageSlug = encodeURIComponent(whosCalling);
+        const matchSlug = [match.date].concat(match.teams).join('/');
+        const endpoint = match.id || matchSlug;
```

Comment:
You are right. I wanted to avoid the ugly formatting, because it doesn't fit on a single line (now it does, thanks to a shorter variable name). Thanks 👍